app/src/main/java/com/shagalalab/constitution/list/adapter/ListAdapter.kt

```diff
@@ -0,0 +1,31 @@
+package com.shagalalab.constitution.list.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.shagalalab.constitution.R
+import com.shagalalab.constitution.list.model.Item
+
+class ListAdapter(private val itemClickListener: ItemClickListener) :
+    RecyclerView.Adapter<ItemViewHolder>() {
+
+    var models: List<Item> = arrayListOf()
```

Comment:
I will merge for now, but pay attention to the warnings from Android studio, here for example it is saying this can become `private`. You can fix it with your next task.